keyboards/lily58/rev1/rev1.h

```diff
@@ -0,0 +1,63 @@
+#ifndef REV1_H
+#define REV1_H
+
+#include "../lily58.h"
```

Comment:
The ".." shouldn't be needed, as the main folder is actually included in the path for the built script